src/airstack/generic.py

```diff
@@ -225,3 +234,465 @@ def remove_unused_variables(document_ast, query):
+            if not nft_exists:
+                _nfts.append({
+                    'name': name,
+                    'image': logo.get('small'),
+                    'blockchain': 'ethereum',
+                    'address': address,
+                    'tokenNfts': token_nft
+                })
+            recommended_users[existing_user_index]['nfts'] = _nfts
+        else:
+            recommended_users.append({
+                **owner,
+                'nfts': [{
+                    'name': name,
+                    'image': logo.get('small'),
+                    'blockchain': 'ethereum',
+                    'address': address,
+                    'tokenNfts': token_nft
+                }]
+            })
+
+    return recommended_users
+
+
+def format_polygon_nft_data(data, _recommended_users=None):
+    if _recommended_users is None:
+        _recommended_users = []
+
+    recommended_users = _recommended_users.copy()
+
+    for nft in data or []:
+        owner = nft.get('owner', {})
+        token = nft.get('token', {})
+
+        name = token.get('name')
+        logo = token.get('logo', {})
+        address = token.get('address')
+        token_nfts = token.get('tokenNfts', [])
+        addresses = owner.get('addresses', [])
+        token_nft = token_nfts[0] if len(token_nfts) > 0 else None
+
+        existing_user_index = -1
+        for index, recommended_user in enumerate(recommended_users):
+            recommended_user_addresses = recommended_user.get('addresses', [])
+            if any(addr in recommended_user_addresses for addr in addresses):
+                existing_user_index = index
+                break
+
+        if existing_user_index != -1:
+            _addresses = recommended_users[existing_user_index].get(
+                'addresses', [])
+            _addresses.extend(addresses)
+            _addresses = list(set(_addresses))  # Remove duplicates
+            recommended_users[existing_user_index]['addresses'] = _addresses
+
+            _nfts = recommended_users[existing_user_index].get('nfts', [])
+            nft_exists = any(nft['address'] == address for nft in _nfts)
+            if not nft_exists:
+                _nfts.append({
+                    'name': name,
+                    'image': logo.get('small'),
+                    'blockchain': 'polygon',
+                    'address': address,
+                    'tokenNfts': token_nfts
+                })
+            recommended_users[existing_user_index]['nfts'] = _nfts
+        else:
+            recommended_users.append({
+                **owner,
+                'nfts': [{
+                    'name': name,
+                    'image': logo.get('small'),
+                    'blockchain': 'polygon',
+                    'address': address,
+                    'tokenNfts': token_nfts
+                }]
+            })
+
+    return recommended_users
+
+
+default_score_map = {
+    'tokenSent': 10,
```

Comment:
The default value config is to kept in another config folder.

Also, better to use environment variables for this too.
https://github.com/Airstack-xyz/airstack-ai-service/blob/main/configurations/conf.py